index.js

```diff
@@ -8,24 +8,23 @@ var TrackerGroup = exports.TrackerGroup = function (name) {
   EventEmitter.call(this)
   this.name = name
   this.trackGroup = []
-  var self = this
   this.totalWeight = 0
-  var noteChange = this.noteChange = function (name) {
-    self.emit("change", name || this.name)
-  }.bind(this)
```

Comment:
O.o I ... uh, I don't know why I was both wrapping `self` AND binding, but that's obviously wrong.